build/get_python_test_deps.py

```diff
@@ -76,10 +102,24 @@ def DownloadWebTestIfNecessary():
   _Download(WEBTEST_URL, webtest_file)
   _Untar(webtest_file)
 
+# Downloads the requests python module used to download other deps.
+def DownloadRequestsIfNecessary():
+  request_file = 'requestsPythonModule'
```

Comment:
Can't we use a more standardized way to download this instead of creating our own way? Can't bower do it for us?